presto-spi/src/main/java/com/facebook/presto/spi/Plugin.java

```diff
@@ -33,4 +33,6 @@ default void setOptionalConfig(Map<String, String> optionalConfig) {}
     {
         return emptyList();
     }
+
+    default void setServerInfo(ServerInfo serverInfo) {}
```

Comment:
We are moving away from this. See #5741 (probably best to rebase on top of that)